cmd/kubeadm/app/util/staticpod/utils.go

```diff
@@ -102,6 +102,28 @@ func NewVolumeMount(name, path string, readOnly bool) v1.VolumeMount {
 	}
 }
 
+// VolumeMapToSlice returns a slice of volumes from a map's values
+func VolumeMapToSlice(volumes map[string]v1.Volume) []v1.Volume {
```

Comment:
unit tests here please (for both instances)

Reply:
will do